src/compiler/gcc.rs

```diff
@@ -469,7 +475,9 @@ where
     }
     // Can't cache compilations with multiple inputs.
     if multiple_input {
-        cannot_cache!("multiple input files");
+        const SEP: &str = ", ";
+        let multiple_input_files_string = multiple_input_files.join(SEP);
+        cannot_cache!("multiple input files", multiple_input_files_string);
```

Comment:
```suggestion
        cannot_cache!("multiple input files", format!("{:?}", multiple_input_files_string));
```

Reply:
Changed. String formatting seems much cleaner in this case;)